tests/rules/no-multiple-actions-in-effects.test.ts

```diff
@@ -8,200 +8,171 @@ import { ruleTester } from '../utils'
-          seven$ = createEffect(() =>
-            this.actions$.pipe(map(() => foo()))
-          )
-        }`,
-      parserOptions: {
-        project: './tsconfig.tests.eslint.json',
-      },
-      filename: 'test.ts',
-    },
-    {
-      code: `
-        export class Effects {
-          five$ = createEffect(() =>
-            this.actions$.pipe(switchMap(() => {
-              return of(foo())
-            }))
-          )
-        }`,
-      parserOptions: {
-        project: './tsconfig.tests.eslint.json',
-      },
-      filename: 'test.ts',
-    },
-    {
-      code: `
-        export class Effects {
-          thirteen$ = createEffect(() =>
-            this.actions$.pipe(mapTo(foo()))
-          )
-        }`,
-      parserOptions: {
-        project: './tsconfig.tests.eslint.json',
-      },
-      filename: 'test.ts',
-    },
-    {
-      code: `
-        export class Effects {
-          eleven$ = createEffect(() =>
-            this.actions$.pipe(switchMapTop([foo()]))
-          )
-        }`,
-      parserOptions: {
-        project: './tsconfig.tests.eslint.json',
-      },
-      filename: 'test.ts',
-    },
-    {
-      code: `
-        export class Effects {
-          twelve$ = createEffect(() =>
-            this.actions$.pipe(aconcatMapTo([bar()]))
-          )
-        }`,
-      parserOptions: {
-        project: './tsconfig.tests.eslint.json',
-      },
-      filename: 'test.ts',
-    },
-    {
-      code: `
-        export class Effects {
-          nine$ = createEffect(() =>
-            this.actions$.pipe(mergeMapTo(foo()))
-          )
-        }`,
-      parserOptions: {
-        project: './tsconfig.tests.eslint.json',
-      },
-      filename: 'test.ts',
-    },
-    {
-      code: `
-        export class Effects {
-          six$ = createEffect(() =>
-            this.actions$.pipe(
-              exhaustMap(() => {
-                return of({}).pipe(
-                  map(response => foo()),
-                  catchError(() => of(bar()))
-                );
-              })
-            )
+    `
+      @Injectable()
+      export class Effects {
+        effectOK$ = createEffect(() =>
+          this.actions$.pipe(map(() => foo()))
+        )
+      }`,
+    `
+      @Injectable()
+      export class Effects {
+        effectOK1$ = createEffect(() =>
+          this.actions$.pipe(switchMap(() => {
+            return of(foo())
+          }))
+        )
+      }`,
+    `
+      const action = () => foo()
+      @Injectable()
+      export class Effects {
+        effectOK2$ = createEffect(() =>
+          this.actions$.pipe(mapTo(action()))
+        )
+      }`,
+    `
+      @Injectable()
+      export class Effects {
+        effectOK3$ = createEffect(() =>
+          this.actions$.pipe(switchMapTop([foo()]))
+        )
+      }`,
+    `
+      @Injectable()
+      export class Effects {
+        effectOK4$ = createEffect(() =>
+          this.actions$.pipe(aconcatMapTo([bar()]))
```

Comment:
```suggestion
          this.actions$.pipe(concatMapTo([bar()]))
```

Reply:
I didn't change this part. I just converted the valid cases to be simple `strings` instead `objects`, because it was unnecessary.

I believe that these cases are like this to ensure we don't flag any custom rxjs operator, so it validates if your `RegExp` is restricted to built-in rxjs operators.

Reply:
Done! Check this out please.

Reply:
Worth mentioning that `rxjs` team plans to deprecate this xMapTo variants as stated here: https://github.com/ReactiveX/rxjs/issues/6399

Reply:
Worth mentioning that `rxjs` team plans to deprecate this `MapTo` variants as stated here: https://github.com/ReactiveX/rxjs/issues/6399.